eth/tracers/native/call.go

```diff
@@ -125,7 +126,7 @@ type callTracerConfig struct {
 
 // newCallTracer returns a native go tracer which tracks
 // call frames of a tx, and implements vm.EVMLogger.
-func newCallTracer(ctx *tracers.Context, cfg json.RawMessage) (*tracers.Tracer, error) {
+func newCallTracer(ctx *tracers.Context, cfg json.RawMessage, _ *params.ChainConfig) (*tracers.Tracer, error) {
```

Comment:
Please don't use unnamed params. It  was an unfortunate language design that the authors also think is bad now but can't revert it. Still, live code should not use it, it just makes things harder to understand and later modify if the param is ever needed.